### PR TITLE
Update Matrix.org-SDKs Documentation

### PR DESCRIPTION
**Pull request by SjxSubham**
"If you are a developer and are not sure what the difference between clients and appservices, please head to the developers documentation."
#2079 issue, (BUG)... i have tried to solve that

### CODE REVIEW

File: content/ecosystem/sdks/index.md [hunk 1]
```diff
@@ -19,6 +19,6 @@ will find below SDKs by type of product you may want to create (namely: clients,
 bots, bridges). Don't be surprised if a same SDK appears in several sections!
 
 If you are a developer and are not sure what the difference between clients and
-appservices, please head to [the developers documentation](/docs/developers/).
+appservices, please head to [the developers documentation](/docs/chat_basics/matrix-for-im/).
```
Comment (thibaultamartin) [inline content/ecosystem/sdks/index.md:22]:
```suggestion
appservices, please head to [the developers documentation](/docs/matrix-concepts/).
```

I believe this is the most appropriate doc

**Review by thibaultamartin — CHANGES_REQUESTED** [review]

File: content/ecosystem/sdks/index.md (hunk 1 above)
Comment (SjxSubham) [reply content/ecosystem/sdks/index.md:22]: Done @thibaultamartin

**Comment by SjxSubham** [comment]
@thibaultamartin I think it's fixed now, kindly review it, if another changes required pls, tell me
issue #2079

**Review by thibaultamartin — APPROVED** [review]
Thanks for you contribution and for helping us keep the docs up to date!

**Comment by SjxSubham** [comment]
closed #2079

**Pull request merged**